magento/magento2
# 33398 fix is_anchor for disabled categories on global level

### Description (*)
I changed products reindexer after category saving to fix issue magento/magento2#33398.
### Fixed Issues (if relevant)

1. Fixes magento/magento2#33398

### Manual testing scenarios (*)
1. Create category "Parent" in Admin store, deactivate it, is anchor = true
2. Create category "Parent -> Child" In Admins store as a child, deactivate it, is anchor = true
3. Now assign a product to The Child category
4. Go to a store view, enable the Parent and the Child Categories on store level
5. Check in the frontend: "Parent"

### Questions or comments


### Contribution checklist (*)
